modules/layers/src/solid-polygon-layer/solid-polygon-layer.js

```diff
@@ -282,7 +282,7 @@ export default class SolidPolygonLayer extends Layer {
           // otherwise it is an Attribute object
           newAttributes[attributeName] = attributeOverride
             ? Object.assign({}, attribute, attributeOverride, {
-                buffer: attribute instanceof Buffer ? attribute : attribute.getBuffer()
+                buffer: attribute.getBuffer()
```

Comment:
Nit: This type checking seems like logic one would prefer not to have in individual layers. Would expect things to eventually be consolidated via `Attribute`?x